tests/playwright-test/ui-mode-trace.spec.ts

```diff
@@ -16,6 +16,9 @@
 
 import { createImage } from './playwright-test-fixtures';
 import { test, expect, retries } from './ui-mode-fixtures';
+import http from 'node:http';
+import httpProxy from 'http-proxy';
+import { ManualPromise } from 'packages/playwright-core/lib/utils/manualPromise';
```

Comment:
We usually use relative imports I think, packages/playwright works by accident here I think.

Reply:
this was a remnant, didn't even use it